packages/mux-player/src/index.ts

```diff
@@ -217,6 +218,24 @@ class MuxPlayerInternal {
     const supportsVolume = await hasVolumeSupportAsync();
     this._setState({ supportsVolume });
   }
+
+  async _setUpPlayerReady(el: MuxPlayerElement) {
+    const { video } = el;
+    if (!video) {
+      console.error("The video element is missing!");
+      return;
+    }
+
+    await Promise.all([
+      // If there is no video src on construction don't be waiting on loadstart.
+      // It might look like we need to check video.networkState here also but
+      // the loadstart event is always fired a tick after the video src is set.
+      video.src && promisify(video.addEventListener, video)("loadstart"),
```

Comment:
blocking: What happens here if we set e.g. `playbackId` after initial load/instantiation of the `mux-player`? Will this still work and work as expected?

Reply:
I think so because there won't be a `src` and the condition will return `false`, 
Promise.all would resolve the `false` immediately.

I'll double check tough to be sure

Reply:
ah do we want playerready to fire immediately when there's no src? I'd assume no.

Reply:
yes, I think we should see this more as an API ready event, ready to take commands from the outside.